### PR TITLE
Create remove

### PR DESCRIPTION
**Pull request by AnselemOdims**
# Steps
- Installed the necessary dependencies
- Configured the redux store
- Created action types and creators
- Included the Provider component and initialized it to the store
- Created a submitBook handler function and dispatched to store
- Created the Book and BooksList components
- Subscribed to the store in the BookList component using the useSelector

# Screenshot
image

### CODE REVIEW

File: src/components/Button.js [hunk 1]
```diff
@@ -1,13 +1,14 @@
 /* eslint-disable react/button-has-type */
```
Comment (arslanbisharat) [inline src/components/Button.js]: - [ ] Don't disable linter rules.

File: src/components/InputBook.js [hunk 2]
```diff
@@ -1,19 +1,44 @@
+/* eslint-disable no-unused-expressions */
```
Comment (arslanbisharat) [inline src/components/InputBook.js]: - [ ] Don't disable linter rules.

File: src/redux/books/books.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
+// Action Types
+const BOOK_ADDED = 'bookStore/books/BOOK_ADDED';
+const BOOK_REMOVED = 'bookStore/books/BOOK_REMOVED';
+
+/**
+ * @function addBook - the action creator for adding a book
+ * @param {Object} payload - the payload for adding a new Book
+ * @returns {Object} - the action object
+ */
```
Comment (arslanbisharat) [inline src/redux/books/books.js]: - [ ] remove commented code

File: src/redux/books/books.js [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+export const addBook = (payload) => ({
+  type: BOOK_ADDED,
+  payload,
+});
+
+/**
+ * @function removeBook - the action creator for removing a book
+ * @param {Object} payload - the payload for removing a Book
+ * @returns {Object} - the action object
+ */
```
Comment (arslanbisharat) [inline src/redux/books/books.js]: remove commented code

File: src/redux/books/books.js [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+
+// declare the initial state
+const initialState = [];
+
+/**
+ * @function booksReducer - the reducer function for books
+ * @param {Array} state - the current state
+ * @param {Object} param1 - the action object
+ * @returns {Array} - the current state array
+ */
```
Comment (arslanbisharat) [inline src/redux/books/books.js]: remove commented code

File: src/redux/configureStore.js [hunk 6]
```diff
@@ -0,0 +1,14 @@
+/* eslint-disable */
```
Comment (arslanbisharat) [inline src/redux/configureStore.js]: - [ ] Don't disbale eslint rules

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
# Changes Requested, 

Hi there, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have limited number of code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: src/components/Button.js (hunk 1 above)
Comment (AnselemOdims) [reply src/components/Button.js]: Ok. Thanks

File: src/redux/books/books.js (hunk 3 above)
Comment (AnselemOdims) [reply src/redux/books/books.js]: Ok, thanks. But it is actually not commented code but JSDoc comment

**Comment by omar-labana** [comment]
Hi @AnselemOdims, 

- [ ] _[OPTIONAL]_  You can return the JS docs comments (but do you really need them?).

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by AnselemOdims** [comment]
Thanks @omar-labana

**Pull request merged**